app/views/templates/errors/errorSummary.scala.html

```diff
@@ -0,0 +1,45 @@
+@*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import uk.gov.hmrc.play.views.html.helpers.{errorSummary => summary}
+
+@(heading: String, form: Form[_], formId: Option[String] = None)(implicit messages: Messages)
```

Comment:
Could you add the `this()` constructor so that this is injectable, and then DI it into the views that use it rather than call it directly as an object.